health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/AssumptionsList.js

```
@@ -1,21 +1,19 @@
+import { Button, Card, LabelFieldPair } from '@egovernments/digit-ui-components';
 import HeaderComp from './HeaderComp';
 import { useTranslation } from 'react-i18next';
+import { EditIcon } from '@egovernments/digit-ui-react-components';
 
 const AssumptionsList = ({ customProps }) => {
     const { t } = useTranslation();
 
-    // Safely access assumptionValues using optional chaining
     const assumptionValues = customProps?.sessionData?.HYPOTHESIS?.Assumptions?.assumptionValues || [];
-
     let dic = {};
```

Biome

[error] This let declares a variable that is only assigned once.

'dic' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)